docs/cugraph/source/index.rst

```diff
@@ -1,13 +1,27 @@
-Welcome to cuGraph's documentation!
-===================================
-RAPIDS cuGraph is a library of graph algorithms that seamlessly integrates
-into the RAPIDS data science ecosystem and allows the data scientist to
-easily call graph algorithms using data stored in GPU DataFrames,
-NetworkX Graphs, or even CuPy or SciPy sparse Matrices.
+Welcome to RAPIDS Graph documentation
+=====================================
+RAPIDS Graph covers a range of libraries and packages, that includes:
+
+__cugraph__: focuses on GPU accelerated graph algorithms.
+
+__cugraph-ops__: focuses on GNN aggregators and operators.
+
+__cugraph-pg__: focuses on providing support for Property Graph (and future Knowledge Graph).
```

Comment:
I don't think this exists yet, is that okay?

Reply:
yep.   That will be discussed in the section on PG and I will be updating this intro section